### ISSUE TITLE
Auto select source 

### ISSUE THREAD

**Issue by ShankarKSH001** [recloudstream/cloudstream#928]
### Describe your suggested feature

Once any source source selected after that when opening different content automatically select source that previously selected like quality and language.
(English is only my first language hope you understand 😅)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by ShankarKSH001** [comment]
> Yeah boy... that's a problem that i recently faced...like if you stream a series using a source which has your regional audio ...in next episode it automatically goes to default

Hope they will update.

**Comment by JimZiii** [comment]
Yes, this already exists, if you find a source you prefer, just increase its prioritization number and it will start automatically next time. Same goes for resolutions
